Support Units: Timeseries/Datapoint Extension (#1392)
cognitedata/cognite-sdk-python: docs/source/core_data_model.rst

Data points
-----------
.. warning::

>     TimeSeries Unit is a new feature:

Reviewer: ```suggestion
    TimeSeries unit support is a new feature:
```

      * The API specification is in beta.
      * The SDK implementation is in alpha.

>     Unit is implemented in the Datapoints APIs with the parameters `target_unt` in

Reviewer: ```suggestion
    Unit conversion is implemented in the Datapoints APIs with the parameters `target_unit` and `target_unit_system` in
```

>     the retrieve methods below. It is only the use of this arguments that is in alpha. Using the methods below

Reviewer: ```suggestion
    the retrieve methods below. It is only the use of these arguments that is in alpha. Using the methods below
```

>     the retrieve methods below. It is only the use of these arguments that is in alpha. Using the methods below
>     without this arguments is stable.

Reviewer 2: ```suggestion
    the retrieve methods below. It is only the use of these arguments that is in alpha. Using the methods below
    without these arguments is stable.
```

Time series
-----------

.. warning::
    TimeSeries unit support is a new feature:
      * The API specification is in beta.
      * The SDK implementation is in alpha.

>     Unit is implemented in the TimeSeris APIs with the parameters `unit_external_id` and `unit_quantity` in
>     the methods below. It is only the use of these arguments that is in alpha. Using the methods below

Reviewer 2: ```suggestion
    Unit is implemented in the TimeSeries APIs with the parameters `unit_external_id` and `unit_quantity` in
    the methods below. It is only the use of these arguments that is in alpha. Using the methods below
```
